src/runtime/virtcontainers/sandbox_test.go

```diff
@@ -771,6 +771,24 @@ func TestSandboxCreateAssets(t *testing.T) {
 		err = createAssets(context.Background(), config)
 		assert.Error(err, msg)
 	}
+
+	// Remote Hypervisor scenario for ImagePath
+	msg := fmt.Sprintf("test[image]: imagePath")
```

Comment:
We are hitting a static checks error here:
```
sandbox_test.go:776:9: S1039: unnecessary use of fmt.Sprintf (gosimple)
	msg := fmt.Sprintf("test[image]: imagePath")
```
I think this would be fine?
```suggestion
	msg := fmt.Sprint("test[image]: imagePath")
```

Reply:
Thanks Steve, incorporated this..

Reply:
Seems like, still it's not happy.
```
msg := fmt.Sprint("test[image]: imagePath")
```
Trying out a simple string var here,
```
msg := "test[image]: imagePath"
```